lib/toolshed.ex

```diff
@@ -44,6 +44,13 @@ defmodule Toolshed do
       import Toolshed
       require Toolshed
 
+      if unquote(Code.ensure_loaded?(Nerves.Runtime) and not Code.ensure_loaded?(Toolshed.Nerves)) do
+        IO.warn("""
+        Nerves-specific helpers have been removed from :toolshed.
+        Add :toolshed_nerves to your Nerves project's dependencies.
```

Comment:
The user will also need to run `use Toolshed.Nerves`. Perhaps say something about adding that line to their `rootfs_overlays/etc/iex.exs` since that would be a common place for it.

Reply:
OK, I improved the warning talking about `rootfs_overlays/etc/iex.exs`.

As of commit d559f2b91a2f849cdc6efc85033755c76208102b,
when a Nerves firmware project does not have `toolshed_nerves` in its dependencies,

this warning message is printed on load:

![](https://user-images.githubusercontent.com/7563926/190916359-abb90ba0-5626-4c00-8ea2-f0cf8a1a836c.png)

Reply:
UPDATES
Instead of letting the users use both `Toolshed` and `Toolshed.Nerves` explicitly, we decided to implicitly use `Toolshed` when `use Toolshed.Nerves` is invoked.

See https://github.com/elixir-toolshed/toolshed_nerves/pull/5